product listing page/products.html

```diff
@@ -0,0 +1,365 @@
+                    </div>
+                  
+                    <div class="product-title m-2t">
+                        Britney Spears Fantasy Eau de Parfum 
+                    </div>
+                    <div class="text-primary m-5">In Stock</div>
+                    <div class="text-secondary m-5"><span style="color:black">Sold by </span>Appario Retail Private
+                        Ltd</div>
+                    <div class="sub-title m-5">
+                        Britney Spears Fantasy Eau de Parfum 
+                    </div>
+                    <div class="price-container m-5">
+                        <div class="price">₹1,699.00</div>
+                    </div>
+                    <div class"delivery details">
+                        <div class="delivery-details m-5"><span style="color:blue">Free delivery: </span> Thursday,
+                            Feb 10 </div>
+                    </div>
+                    <div class=" card-btn-container m-5">
+                        <button class=" btn-product btn-product btn-padding">
+                            <i class="fas fa-shopping-cart" aria-hidden="true"></i>
+                            <a href="card.html">Add to Cart</a>
+                        </button>
+                        <button class="btn btn-secondary btn-padding">
+                            <a href="card.html">Save for later</a>
+                        </button>
+
+                    </div>
+                </div>
+                <!-- 5card -->
+                <div class="card-product padding">
+                    <div class="card-image">
+                        <img class="image-product" src="/assests/men category.png"
+                            alt="product image" />
+                    </div>
+                  
+                    <div class="product-title m-2t">
+                        Britney Spears Fantasy Eau de Parfum 
+                    </div>
+                    <div class="text-primary m-5">In Stock</div>
+                    <div class="text-secondary m-5"><span style="color:black">Sold by </span>Appario Retail Private
+                        Ltd</div>
+                    <div class="sub-title m-5">
+                        Britney Spears Fantasy Eau de Parfum 
+                    </div>
+                    <div class="price-container m-5">
+                        <div class="price">₹1,699.00</div>
+                    </div>
+                    <div class"delivery details">
+                        <div class="delivery-details m-5"><span style="color:blue">Free delivery: </span> Thursday,
+                            Feb 10 </div>
+                    </div>
+                    <div class=" card-btn-container m-5">
+                        <button class=" btn-product btn-product btn-padding">
+                            <i class="fas fa-shopping-cart" aria-hidden="true"></i>
+                            <a href="card.html">Add to Cart</a>
```

Comment:
I think a tag should not be required because this button won't be redirecting to anther link

Reply:
yes you are right